### PR TITLE
Updated jetbrains/phpstorm-stubs

### PR DESCRIPTION
**Pull request by kukulich**

### CODE REVIEW

File: bin/generate-phpstorm-stubs-map.php [hunk 1]
```diff
@@ -1,243 +0,0 @@
-<?php
```
Comment (Ocramius) [inline bin/generate-phpstorm-stubs-map.php]: 🔥

File: src/SourceLocator/SourceStubber/PhpStormStubsMap.php [hunk 2]
```diff
@@ -1,10885 +0,0 @@
-<?php
```
Comment (Ocramius) [inline src/SourceLocator/SourceStubber/PhpStormStubsMap.php]: 🔥

**Comment by Ocramius** [comment]
CS failed due to config:

```
ERROR: The file "bin" does not exist.
```

**Comment by kukulich** [comment]
@Ocramius Finally. However, I really don't know how I've fixed it :)

**Comment by Ocramius** [comment]
Thanks @kukulich and @WyriHaximus!

**Comment by WyriHaximus** [comment]
> @Ocramius Finally. However, I really don't know how I've fixed it :)

Black magic most likely, thanks!

**Pull request merged**